lib/tasks/javascript.rake

```diff
@@ -74,6 +74,7 @@ def dependencies
     { source: "chart.js/dist/chart.min.js", public: true },
     { source: "chartjs-plugin-datalabels/dist/chartjs-plugin-datalabels.min.js", public: true },
     { source: "diffhtml/dist/diffhtml.min.js", public: true },
+    { source: "morphlex/dist/morphlex.js", public: true },
```

Comment:
I think the "new" way of adding dependencies is via webpack. I think we did `jsuites` that way recently, see https://github.com/discourse/discourse/pull/25018

Reply:
Thanks 👍

Reply:
Yup exactly 💯 

Bonus points for using async `import()` so that it only gets loaded when the composer is opened!

Reply:
It's still a WIP, but is that what you had in mind when you said "async `import()`" @davidtaylorhq ?